client/main.js

```diff
@@ -17,6 +17,11 @@ $(document).ready(function () {
     let genres = []
     let genrePreferences = [];
     let decadePreferences = [];
+    let settingsDisplay = $('#advSetForm');
+    let stopDisplay = $('#stop-timer');
+    let startDisplay = $('#start-timer');
+    let volumeDisplay = $('#vol-control');
+    let titleDisplay = $('#current-song-display');
```

Comment:
What's the point of these, if they're not even used?

Reply:
I definitely use these